models/predicates.go

```diff
@@ -94,3 +98,5 @@ var organizationTypes = []string{
 	"http://vivoweb.org/ontology/core#StudentOrganization",
 	"http://vivoweb.org/ontology/core#University",
 }
+
+var projectType = "http://xmlns.com/foaf/0.1/Project"
```

Comment:
I don't feel strongly about this, but what would you think of renaming and retyping this variable like so:

```golang
var projectTypes = []string{
        "http://xmlns.com/foaf/0.1/Project",
}
```

That way, 1) if there are new project types, we can add those URIs to the list here without changing code elsewhere, and 2) all types look alike.

Reply:
Sure.